### PR TITLE
NMS-12497: migrate docs to antora

### PR DESCRIPTION
**Pull request by Bonrob2**
updated broken links.

# Pull Request Check Sheet

* [ ] Have you read and followed our [Contribution Guidelines](https://github.com/OpenNMS/opennms/blob/develop/CONTRIBUTING.md)?
* [ ] Have you [made an issue in the OpenNMS issue tracker](https://issues.opennms.org)?<br>If so, you should:
  1. update the title of this PR to be of the format: `${JIRA-ISSUE-NUMBER}: subject of pull request`
  2. update the JIRA link at the bottom of this comment to refer to the real issue number
  3. prefix your commit messages with the issue number, if possible
* [ ] Have you made a comment in that issue which points back to this PR?
* [ ] Have you updated the JIRA link at the bottom of this comment to link to your issue?
* [ ] If this is a new feature, is there documentation?
* [ ] If this is a new feature or substantial change, are there tests that cover it?

# Pull Request Process

One or more reviewers should be assigned to each PR.

If you know that a particular person is subject matter expert in the area your PR affects, feel free to assign one or more reviewers when you create this PR, otherwise reviewers will be assigned for you.

Once the reviewer(s) accept the PR and the branch passes continuous integration in Bamboo, the PR is eligible for merge.

At that time, if you have commit access (are an OpenNMS Group employee or a member of the Order of the Green Polo) you are welcome to merge the PR.
Otherwise, a reviewer can merge it for you.

Thanks for taking time to contribute!

# External References

* JIRA (Issue Tracker): http://issues.opennms.org/browse/NMS-12497
* Continuous Integration: [Bamboo](https://bamboo.opennms.org/), [CircleCI](https://circleci.com/gh/OpenNMS/opennms-js)

### CODE REVIEW

**Comment by Bonrob2** [comment]
@indigo423 found two broken links in the Helm docs.

File: docs/modules/panel_configuration/pages/dynamic-dashboard.adoc [hunk 1]
```diff
@@ -1,7 +1,7 @@
 :imagesdir: ../assets/images
 [.lead]
 
-[[pc-dyanmic-dash]]
+[[pc-dynanmic-dash]]
```
Comment (RangerRick) [inline docs/modules/panel_configuration/pages/dynamic-dashboard.adoc:4]: ...dyna**n**mic?  :D

Comment (Bonrob2) [reply docs/modules/panel_configuration/pages/dynamic-dashboard.adoc:4]: :-D Yes. sigh.

**Review by indigo423 — APPROVED** [review]
LGTM! 👍

**Pull request merged**